packages/core/src/components/CopyTextButton/CopyTextButton.tsx

```diff
@@ -44,23 +44,36 @@ const useStyles = makeStyles(theme => ({
  * Example:
  *    <CopyTextButton text="My text that I want to be copied to the clipboard" />
  */
-const CopyTextButton = ({
-  text,
-  tooltipDelay = 1000,
-  tooltipText = 'Text copied to clipboard',
-}) => {
-  const classes = useStyles();
+type Props = {
+  text: string;
+  tooltipDelay?: number;
+  tooltipText?: string;
+};
+
+const defaultProps = {
```

Comment:
Curious about why you prefer this? No strong opinions though.

Reply:
Are you asking about my preference for the default values of the props (which was there already) or this way of handling the default props?

I looked at how other components handle default props and found this way in `CircleProgress`. It looks neat, so I thought I should implement in the same way (for consistency). :)

https://github.com/spotify/backstage/blob/af6dcfff5deb81ead7fc6918cee4b76340e7dbec/packages/core/src/components/ProgressBars/CircleProgress.tsx#L51-L56
https://github.com/spotify/backstage/blob/af6dcfff5deb81ead7fc6918cee4b76340e7dbec/packages/core/src/components/ProgressBars/CircleProgress.tsx#L78-L81

Reply:
The way of handling them. And ok, clear enough :grin:

Reply:
Sometimes I feel GitHub is way behind in the emoji game. I want to react with party parrot, but I can't :/